src/core/defs.jl

```diff
@@ -187,6 +187,24 @@ dim_value_dict(md::ModelDef) = Dict([name => collect(values(dim)) for (name, dim
 Base.haskey(md::ModelDef, name::Symbol) = haskey(md.dimensions, name)
 
 isuniform(md::ModelDef) = md.is_uniform
+
+
+# Helper function invoked when the user resets the time dimension with set_dimension!
+# This function calls set_run_period on each component definition to reset the first and last values.
```

Comment:
tiny fix to the description ... correct function name above to `set_run_period!`

Reply:
will do!